data/json/npcs/refugee_center/surface_staff/Smokes/free_merchant_shopkeep_missions.json

```diff
@@ -0,0 +1,360 @@
+      "accepted": "If you can do this for us our survival options would vastly increase.",
+      "rejected": "Come back when you get a chance, we really need to start reclaiming the region.",
+      "advice": "Solar panels are pretty common on electric cars and on rooftops.",
+      "inquire": "How is the search going?",
+      "success": "Great, I know it isn't much but we hope to continue to expand thanks to your help.",
+      "success_lie": "What good does this do us?",
+      "failure": "It was a lost cause anyways…"
+    }
+  },
+  {
+    "id": "MISSION_FREE_MERCHANTS_EVAC_5",
+    "type": "mission_definition",
+    "name": { "str": "In a Pickle" },
+    "description": "Deliver 100 half-liter glass canning jars to the Free Merchants to kickstart their food storage plans.",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 3,
+    "value": 400000,
+    "item": "jar_glass_sealed",
+    "count": 100,
+    "end": {
+      "effect": [
+        { "u_spawn_item": "FMCNote", "count": 25 },
+        { "u_add_var": "FMShopkeep_Mission5", "type": "mission", "context": "flag", "value": "yes" }
+      ]
+    },
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "dialogue": {
+      "describe": "We need help…",
+      "offer": "While we've managed to secure day to day food supplies, it's very much hand to mouth.  Our reserves would barely last a few days if something kept our people locked in.  We need a stockpile to avoid that.  Thanks to our outpost we have a bit of meat and vegetables coming in, but we need a better way to preserve them.  Some of our people know enough about food canning that if we had a good stock of canning jars, we could make pickles and canned meats to get us set for the winter.  I'll pay you a premium rate if you can bring us around a hundred jars to get us started.",
+      "accepted": "It'll take a load off my shoulders if I can finally stop worrying about this.",
+      "rejected": "We'll hold for the moment, but I'll get an ulcer if I have to keep worrying.  If you reconsider let me know.",
+      "advice": "Grocery stores, house kitchens, there's plenty of places to look.",
+      "inquire": "How is the search going?",
+      "success": "That's one less thing to worry about.  Glad to have someone like you on our side.  Here's 25 merch, and the broker will buy any further glass jars you can find - at a reduced price, we can't keep up the premium rate.  Actually, the broker was saying they had some jobs like this for you, and that it might be easier to just go straight through their office if you want more work.",
+      "success_lie": "What good does this do us?",
+      "failure": "It was a lost cause anyways…"
+    }
+  },
+  {
+    "id": "TALK_FREE_MERCHANTS_MERCHANT_Robofac",
```

Comment:
Of note - not for this PR - I believe hub01 spawns with free merchants already present and waiting to trade. In a subsequent PR it would be worth removing those traders, and adding them via update_mapgen after this quest line is done.

Reply:
Right now, the merchants only spawn after completing the quest `Steal a dead man's mind`. I could probably put in a condition that prevents the Free Merchants from giving the breadcrumb quest if the player has already completed this quest for the Hub, though.l

Reply:
Oh cool. Yeah, disabling the breadcrumb quest if you've already made it that far with the hub is a good idea, then later (you've got enough this PR) we can add some new dialogue to replace it.

Reply:
I'll do that in the next commit, along with the ground cover stuff and possibly a change to the search radius on the breadcrumb.